### DISCUSSION TITLE
can plot.variable plot the partial dependence of survival for different stage?

### DISCUSSION THREAD

**Discussion by wymin576** [kogalur/randomForestSRC#447] [Q&A]
hello,
I know the function plot.variable can give partial dependence plots. My question is: Can the function plot.variable give the partial dependence of survival for different stages, such as one-year, five-year, and ten-year?
Any help will be appreciated!

**Comment by ishwaran** [comment]
Yes, it is possible to do that.

Here's an illustration using the veteran data with survival partial effects on the vertical axis for 200, 400, 1000 days.

```
data(veteran, package = "randomForestSRC")
o <- rfsrc(Surv(time,status)~., veteran)
plot.variable(o, surv.type = "surv", xvar.names="karno", time = 200)
plot.variable(o, surv.type = "surv", xvar.names="karno", time = 400)
plot.variable(o, surv.type = "surv", xvar.names="karno", time = 1000)
```

**Comment by wymin576** [comment]
Thank you so much for your reply! your answer solved my problem.
